src/plugins/vis_type_vislib/public/vislib/types.ts

```diff
@@ -33,6 +33,7 @@ export interface TableParent {
   column: number;
   row: number;
   key: number;
+  formattedKey: string | number | boolean;
```

Comment:
`splitColumnFormatter.convert` returns string all times. Not sure that formatted value should has `number`, `boolean` type 
![image](https://user-images.githubusercontent.com/20072247/101916144-993f4500-3bd7-11eb-8ce7-36a4beb005b6.png)

Reply:
Changed to string